lib/auth/webauthn/session.go

```diff
@@ -39,21 +39,40 @@ const scopeLogin = "login"
 // that use in-memory storage.
 const scopeSession = "registration"
 
-func sessionToPB(sd *wan.SessionData) (*wanpb.SessionData, error) {
+// SessionData is a clone of [protocol.SessionData], materialized here to keep a
+// stable JSON marshal/unmarshal representation and add extensions.
+type SessionData struct {
```

Comment:
Move it to lib/auth/webauthntypes, so we avoid eventual cycles/importing the entirety of server-side webauthn just for this type?